docs/src/quickstart.md

```diff
@@ -0,0 +1,93 @@
+# Quickstart
+
+## Writing your first kernel
+
+Kernel functions are marked with the [`@kernel`](@ref). Inside the `@kernel` macro
+you can use the [kernel language](@ref api_kernel_language). As an example, the `mul2`` kernel
+below will multiply each element of the array `A` by `2`. It uses the [`@index`](@ref) macro
+to obtain the global linear index of the current work item.
+
+```julia
+@kernel function mul2_kernel(A)
+  I = @index(Global)
+  A[I] = 2 * A[I]
+end
+```
+
+## Launching kernel on the host
+
+You can construct a kernel for a specific backend by calling the kernel
+function, with the first argument being the device of type `KA.Device`, the
```

Comment:
```suggestion
```suggestion
You can construct a kernel for a specific backend by calling the kernel like so: `mul2_kernel(CPU(), 16)`. Here, the first argument being the device of type `KA.Device`, the
```

I think it flows better with some code. It also helps explain that the configuration is definitely different than the launch event.

Reply:
Reworked this section according to your suggestion.